docs/.vuepress/tools/snippet-transform/templates/index.js

```diff
@@ -0,0 +1,27 @@
+const { render: reactRender } = require('./react');
+
+const TEMPLATE_RENDERERS = {
+  react: reactRender
+};
+
+/**
+ * Render the snippet using a template.
+ *
+ * @param {string} framework The desired framework name.
+ * @param {object} snippetInformation The snippet content.
+ * @param {boolean} [includeImports=false] `true` if the template should include the imports section.
+ * @param {boolean} [includeApp=false] `true` if the template should include the app-wrapping function.
+ * @param {string} [appContainerId] The id of a container to mount Handsontable in. Defaults to `example`.
+ * @returns {string|null}
+ */
+function renderTemplate(framework, snippetInformation, includeImports = false, includeApp = false, appContainerId) {
+  return (
+    typeof TEMPLATE_RENDERERS[framework] === 'function' ?
```

Comment:
Extra `if` and returning `null` would be more clear for me. Using parenthesis for me make it unintuitive.